packages/client/modules/email/components/EmailNotifications/EmailResponseReplied.tsx

```diff
@@ -0,0 +1,95 @@
+import graphql from 'babel-plugin-relay/macro'
+import {convertFromRaw, Editor, EditorState} from 'draft-js'
+import editorDecorators from 'parabol-client/components/TaskEditor/decorators'
+import {EmailResponseReplied_notification$key} from 'parabol-client/__generated__/EmailResponseReplied_notification.graphql'
+import React, {useMemo, useRef} from 'react'
+import {useFragment} from 'react-relay'
+import {cardShadow} from '../../../../styles/elevation'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {FONT_FAMILY} from '../../../../styles/typographyV2'
+import makeAppURL from '../../../../utils/makeAppURL'
+import {notificationSummaryUrlParams} from '../NotificationSummaryEmail'
+import EmailNotificationTemplate from './EmailNotificationTemplate'
+
+const editorStyles = {
+  backgroundColor: '#FFFFFF',
+  borderColor: PALETTE.SLATE_400,
+  borderRadius: '4px',
+  borderStyle: 'solid',
+  borderWidth: '1px',
+  color: PALETTE.SLATE_700,
+  fontFamily: FONT_FAMILY.SANS_SERIF,
+  fontSize: '14px',
+  boxShadow: cardShadow,
+  lineHeight: '20px',
+  margin: '4px 0 0',
+  padding: 8
+}
+
+interface Props {
+  notificationRef: EmailResponseReplied_notification$key
+  appOrigin: string
+}
+
+const EmailResponseReplied = (props: Props) => {
+  const {notificationRef, appOrigin} = props
+  const notification = useFragment(
+    graphql`
+      fragment EmailResponseReplied_notification on NotifyResponseReplied {
+        ...EmailNotificationTemplate_notification
+        author {
+          rasterPicture
+          preferredName
+        }
+        meeting {
+          id
+          name
+        }
+        comment {
+          content
+        }
+      }
+    `,
+    notificationRef
+  )
+  const {meeting, author, comment} = notification
+  const {rasterPicture: authorPicture, preferredName: authorName} = author
+
+  const {id: meetingId, name: meetingName} = meeting
+
+  const linkUrl = makeAppURL(appOrigin, `/meet/${meetingId}`, {
+    searchParams: notificationSummaryUrlParams
```

Comment:
This will incorporate the changes in https://github.com/ParabolInc/parabol/pull/7546/files#diff-62d451086015e1999dc120842420837f8de6d40c4391dc67d772c5a49cfbd0adR45 once that PR lands